services/web-app/components/mdx-notification/mdx-notification.module.scss

```
@@ -43,34 +58,50 @@
   @include type.type-style('heading-compact-01', true);
 
   padding-bottom: spacing.$spacing-01;
+  color: theme.$text-primary;
 }
 
 .link {
   position: absolute;
-  right: 0;
-  padding: 0 spacing.$spacing-07 0 spacing.$spacing-05;
+  top: 6px;
+  right: 1px;
```

```suggestion
  top: convert.rem(8px);
  right: convert.rem(8px);
```

Ghost button spacing fix